### ISSUE TITLE
Implement paging

### ISSUE THREAD

**Issue by Roemer** [Daniel15/simple-nuget-server#27]
When using list, nuget automatically tries to page with a pagesize of 30.
As paging is not implemented yet, it will output only 30 results and then endlessly loop with calls like:

```
GET http://myserver/Search()?$orderby=Id&searchTerm=''&targetFramework=''&includePrerelease=false&$skip=30&$top=30&semVerLevel=2.0.0
OK http://myserver/Search()?$orderby=Id&searchTerm=''&targetFramework=''&includePrerelease=false&$skip=30&$top=30&semVerLevel=2.0.0 40ms 
```

Can be simply reproduced by calling:
`nuget list -AllVersions -Verbosity detailed -Source http://myserver`

**Comment by Daniel15** [comment]
Good catch. Would you like to send a pull request to improve this?

**Comment by Daniel15** [comment]
Fixed in #35
